scrapers/TakeVan.yml

```diff
@@ -0,0 +1,28 @@
+name: "TakeVan"
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - takevan.com/video/
+    scraper: sceneScraper
+
+xPathScrapers:
+  sceneScraper:
+    scene:
+      Title: //h1[@class="videoTitle"]
+      Date: &dateAttr
+        selector: //div[@id="videoDate"]
+        postProcess:
+          - replace:
+              - regex:  "^Added "
+                with: ""
+          - parseDate: January 2, 2006
+      Details: &detailsSel //div[@class="videoDescription"]/text()
+      Tags:
+        Name: //div[@class="videoTags"]/a/text()
+      #Performers:
+      #  Name: $info//div[starts-with(@class,"info")]/a/text()
```

Comment:
```yaml
      Performers:
        Name:
          selector: //div[@class="videoFeaturedModels"]/text()
          split: ", "
```
Adds the performers when mentioned (single named performers are not that useful but some users may want to add them ?)

Reply:
Added :)